packages/drivers/odsp-driver-definitions/src/factory.ts

```diff
@@ -64,6 +64,7 @@ export interface IOpsCachingPolicy {
  */
 export interface ICollabSessionOptions {
 	/**
+	 * @deprecated No longer needed
```

Comment:
I think you wanted to deprecate the other option: forceAccessTokenViaAuthorizationHeader and not this one?

Reply:
I want to deprecate both :)